schema/codelists/tenderStatus.csv

```diff
@@ -1,8 +1,8 @@
-Code,Title,Description
-planning,Planning,"A future contracting process is being considered. Early information about the process can be provided in the tender section. A process with this status might provide information on early engagement or consultation opportunities, during which the details of a subsequent tender can be shaped."
-planned,Planned,"A contracting process is scheduled, but is not yet taking place. Details of the anticipated dates can be provided in the tender block."
-active,Active,A tender process is currently taking place.
-cancelled,Cancelled,The tender process has been cancelled.
-unsuccessful,Unsuccessful,The tender process was unsuccessful.
-complete,Complete,The tender process is complete.
-withdrawn,Withdrawn,No further information on this process is available under this ocid.
+Code,Title,Description,Deprecated,Deprecation note
+planning,Planning,"A future contracting process is being considered. Early information about the process can be provided in the tender section. A process with this status might provide information on early engagement or consultation opportunities, during which the details of a subsequent tender can be shaped.",1.2,"Deprecated as unnecessary, since contracting processes are separate from planning processes (which do not have any states)."
+planned,Planned,"The contracting process is scheduled, but contracting documents (for example, procurement documents) are not yet available to potential suppliers.",1.2,Deprecated because the same information can be determined by the absence of `tender.datePublished` (new in 1.2).
+active,Active,"The contracting documents (for example, procurement documents) are available to potential suppliers, and the bid submission deadline has not yet passed.",1.2,Deprecated because the same information can be determined by comparing `tender.datePublished` (new in 1.2) and `tender/tenderPeriod/endDate` to `date`.
```

Comment:
```suggestion
active,Active,"The contracting documents (for example, procurement documents) are available to potential suppliers, and the bid submission deadline has not yet passed.",1.2,Deprecated because the same information can be determined by the presence of `tender.datePublished` (new in 1.2) and by comparing `tender/tenderPeriod/endDate` to `date`.
```